revivelib/core.go

```diff
@@ -173,14 +175,42 @@ func getPackages(includePatterns []string, excludePatterns ArrayFlags) ([][]stri
 		globs = append(globs, ".")
 	}
 
-	packages, err := dots.ResolvePackages(globs, normalizeSplit(excludePatterns))
+	globs, skips, err := prepareSkips(globs, normalizeSplit(excludePatterns))
+	if err != nil {
+		return nil, fmt.Errorf("prepare skips - resolving excludes before dots: %w", err)
+	}
+
+	packages, err := dots.ResolvePackages(globs, skips)
 	if err != nil {
 		return nil, fmt.Errorf("getting packages - resolving packages in dots: %w", err)
 	}
 
 	return packages, nil
 }
 
+func prepareSkips(globs, excludes []string) ([]string, []string, error) {
+	var skips []string
+	for _, path := range globs {
+		var basepath string
+		basepath, _ = doublestar.SplitPattern(path)
+		fsys := os.DirFS(basepath)
+		for _, skip := range excludes {
+			matches, err := doublestar.Glob(fsys, skip)
+			if err != nil {
+				return nil, nil, fmt.Errorf("Skips Error: %v", err)
```

Comment:
I guess, we should use `%w`.

```suggestion
				return nil, nil, fmt.Errorf("Skips Error: %w", err)
```